packages/fxa-settings/src/pages/Signin/SigninUnblock/container.tsx

```diff
@@ -88,8 +87,7 @@ const SigninUnblockContainer = ({
       });
       return { data };
     } catch (error) {
-      // TODO consider additional error handling - any non-gql errors will return an unexpected error
-      return handleGQLError(error);
+      return getHandledError(error);
```

Comment:
I only improved this for signin pages, it didn't look like we needed it for signup. It's at least there now if we want to use it anywhere else.